### PR TITLE
[WFLY-14709] Upgrade WF Preview's Glassfish JAXB to 3.0.0 and Istack …

### PR DESCRIPTION
**Pull request by bstansberry**
…to 4.0.0

https://issues.redhat.com/browse/WFLY-14709

### CODE REVIEW

**Review by ropalka — APPROVED** [review]
LGTM

**Review by scottmarlow — APPROVED** [review]

**Comment by bstansberry** [comment]
EE 9 Tests:

https://ci.wildfly.org/viewLog.html?buildId=252719&tab=buildResultsDiv&buildTypeId=WF_Ee9BootableJarLinuxJdk8
https://ci.wildfly.org/viewLog.html?buildId=252730&tab=buildResultsDiv&buildTypeId=WF_WildFlyPreviewLinuxJdk11

File: ee-9/feature-pack/src/main/resources/modules/system/layers/base/com/sun/xml/bind/main/module.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        <artifact name="${org.glassfish.jaxb:jaxb-xjc}"/>
+        <artifact name="${org.glassfish.jaxb:xsom}"/>
+        <artifact name="${org.glassfish.jaxb:txw2}"/>
+    </resources>
+
+    <dependencies>
+        <module name="java.logging"/>
+        <module name="java.xml"/>
+        <module name="javax.activation.api"/>
+        <module name="javax.api"/>
```
Comment (boris-unckel) [inline ee-9/feature-pack/src/main/resources/modules/system/layers/base/com/sun/xml/bind/main/module.xml:47]: The other modules are already fine grained. Is this depracted one really necessary? Shall I help to check the jars?

**Comment by bstansberry** [comment]
@boris-unckel This isn't actually workable, as CXF won't work properly with it, so I'm going to close it.

**Pull request closed without merging**